.prettierrc.json

```diff
@@ -0,0 +1,8 @@
+{
+   "trailingComma": "all",
```

Comment:
What does prettier provide that eslint does not? Genuinely curious

Reply:
Prettier only cares about formatting, whereas eslint cares about code-quality rules. Prettier has a good explanation here: https://prettier.io/docs/en/comparison.html